worldbank/src/Components/LogInPage/LogInPage.js

```diff
@@ -38,68 +40,79 @@ class LogInPage extends React.Component {
   getLogInForm = () => {
     return (
       <>
```

Comment:
Do we need these empty divs if it is wrapped already by ThemeProvider? Im not sure but if they are redundant then we should remove them.
Otherwise go ahead and merge. Thanks for sorting the css!

Reply:
They're there to keep the flex in columns rather than rows, but there was one empty div that had no purpose that I removed, thanks for spotting!